aliasmgr/aliasmgr.go

```diff
@@ -220,12 +235,22 @@ func (m *Manager) populateMaps() error {
 // AddLocalAlias adds a database mapping from the passed alias to the passed
 // base SCID. The gossip boolean marks whether or not to create a mapping
 // that the gossiper will use. It is set to false for the upgrade path where
-// the feature-bit is toggled on and there are existing channels.
+// the feature-bit is toggled on and there are existing channels. The linkUpdate
+// flag is used to signal whether this function should also trigger an update
+// on the htlcswitch scid alias maps.
 func (m *Manager) AddLocalAlias(alias, baseScid lnwire.ShortChannelID,
-	gossip bool) error {
+	gossip, linkUpdate bool) error {
```

Comment:
😔😔 Are we committed to this approach?

Reply:
That's how I interpret [this comment](https://github.com/lightningnetwork/lnd/pull/8960#discussion_r1727547252) and [your response](https://github.com/lightningnetwork/lnd/pull/8960#discussion_r1727674945), yes.
Your objections have been noted.

Reply:
Yeah my bad I had forgotten about the resolution on that convo. Disregard.